js/renovation/ui/editors/check_box.tsx

```diff
@@ -149,6 +155,23 @@ export class CheckBox extends JSXComponent(CheckBoxProps) {
     return undefined;
   }
 
+  @Effect()
+  updateIconFontSize(): EffectReturn {
+    const iconElement = this.iconRef?.current;
+    const { iconWidth, iconHeight } = this.props;
+
+    if (iconElement !== null && iconElement !== undefined) {
+      const width = typeof iconWidth === 'number' ? iconWidth : iconElement.offsetWidth;
+      const height = typeof iconHeight === 'number' ? iconHeight : iconElement.offsetHeight;
+      const iconFontSizeRatio = isMaterial(current()) ? 16 / 18 : 16 / 22;
```

Comment:
Where do these magic numbers come from? How do we know that we will have a mismatch between the code and the sizes in scss?

Reply:
it's old `checkbox-arrow-icon-size` and `checkbox-size` for [generic](https://github.com/DevExpress/DevExtreme/blob/21_2/scss/widgets/generic/checkBox/_sizes.scss) and [material](https://github.com/DevExpress/DevExtreme/blob/21_2/scss/widgets/material/checkBox/_sizes.scss) themes